.github/workflows/documentation.yml

```diff
@@ -55,13 +55,22 @@ jobs:
         env:
           EARTHDATA_USERNAME: ${{ secrets.EDL_USERNAME }}
           EARTHDATA_PASSWORD: ${{ secrets.EDL_PASSWORD }}
-
-      - name: Deploy
-        if: |
-          github.event_name == 'push'
-          && (github.ref == 'refs/heads/main' || github.ref == 'ref/heads/documentation')
-          && github.repository == 'nsidc/earthaccess'
-        uses: peaceiris/actions-gh-pages@v3
+      - name: Upload site artifact
+        uses: actions/upload-pages-artifact@v1
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./site
+          path: ./site
+
+  deploy-documentation:
+    if: |
+      github.event_name == 'push'
+      && (github.ref == 'refs/heads/main' || github.ref == 'ref/heads/documentation')
```

Comment:
Where's the `documentation` branch coming from?

Reply:
I don't know, honestly. It was based on stuff that was already here. @betolink ?

Reply:
oh yeah, originally I was thinking about having that branch to test changes to the docs without having to deploy main.